### PR TITLE
Updates www redirects for various projects

### PR DESCRIPTION
**Pull request by dorothyyeager**
Updates www redirects (needed because a broader redirect to transition is overriding Wagtail redirects):
Admin fines pages - https://github.com/fecgov/fec-cms/issues/3783
Enforcement materials page - https://github.com/fecgov/fec-cms/issues3827
Election result directories and pages - https://github.com/fecgov/fec-cms/issues3827

For @patphongs I have a question about doing the broader redirect for the election result directories - there's one for each year. Wasn't sure how to format that so I followed what I did in the transition PR?

### CODE REVIEW

**Comment by dorothyyeager** [comment]
@kathycarothers This is ready for you to review from content team perspective.

**Comment by kathycarothers** [comment]
@dorothyyeager The redirects look good.

File: redirects-www.conf [hunk 1]
```diff
@@ -39,6 +39,7 @@ rewrite ^/pages/record.shtml https://www.fec.gov/updates/record-archive-1975-200
 rewrite ^/pubrec/cfsdd/cfsdd.shtml https://www.fec.gov/introduction-campaign-finance/how-to-research-public-records/combined-federalstate-disclosure-and-election-directory/ redirect;
 rewrite ^/pubrec/cfsdd/ https://www.fec.gov/introduction-campaign-finance/how-to-research-public-records/combined-federalstate-disclosure-and-election-directory/ redirect;
 rewrite ^/pubrec/cfsdd/cfsded_000.pdf https://www.fec.gov/resources/cms-content/documents/cfsded.pdf redirect;
+rewrite ^/pubrec/electionresults.shtml https://www.fec.gov/introduction-campaign-finance/election-and-voting-information/ redirect;
```
Comment (patphongs) [inline redirects-www.conf:42]: Remove and add to Wagtail redirects

File: redirects-www.conf [hunk 2]
```diff
@@ -127,10 +127,16 @@ rewrite ^/law/legalconsideration.shtml https://transition.fec.gov/law/legalconsi
 rewrite ^/em/em.shtml /legal-resources/enforcement/ redirect;
 rewrite ^/em/enfpro/EnforcementProfile.shtml https://transition.fec.gov/em/enfpro/EnforcementProfile.shtml redirect;
 rewrite ^/em/respondent_guide.pdf https://www.fec.gov/resources/cms-content/documents/respondent_guide.pdf redirect;
-rewrite ^/af/af.shtml https://transition.fec.gov/af/af.shtml redirect;
+rewrite ^/af/af.shtml https://www.fec.gov/legal-resources/enforcement/administrative-fines/ redirect;
```
Comment (patphongs) [inline redirects-www.conf:130]: Remove and add to Wagtail redirects

File: redirects-www.conf [hunk 3]
```diff
@@ -127,10 +127,16 @@ rewrite ^/law/legalconsideration.shtml https://transition.fec.gov/law/legalconsi
...
 rewrite ^/pages/brochures/admin_fines.shtml /legal-resources/enforcement/administrative-fines/ redirect;
-rewrite ^/af/AFPRegulations.shtml https://transition.fec.gov/af/AFPRegulations.shtml redirect;
+rewrite ^/af/AFPRegulations.shtml https://sers.fec.gov/fosers/showpdf.htm?docid=351065 redirect;
```
Comment (patphongs) [inline redirects-www.conf:132]: Remove and add to Wagtail redirects

File: redirects-www.conf [hunk 4]
```diff
@@ -127,10 +127,16 @@ rewrite ^/law/legalconsideration.shtml https://transition.fec.gov/law/legalconsi
...
 rewrite ^/af/af_calc.shtml https://www.fec.gov/legal-resources/enforcement/administrative-fines/calculating-administrative-fines/ redirect;
+rewrite ^/af/afcalc_20130725.shtml https://www.fec.gov/legal-resources/enforcement/administrative-fines/calculating-administrative-fines/#calculating-a-fine redirect;
+rewrite ^/af/afcalc_20160701.shtml https://www.fec.gov/legal-resources/enforcement/administrative-fines/calculating-administrative-fines/#calculating-a-fine redirect;
+rewrite ^/af/afcalc_post-200907.shtml https://www.fec.gov/legal-resources/enforcement/administrative-fines/calculating-administrative-fines/#calculating-a-fine redirect;
```
Comment (patphongs) [inline redirects-www.conf:136]: Remove and add to Wagtail redirects

File: redirects-www.conf [hunk 5]
```diff
@@ -127,10 +127,16 @@ rewrite ^/law/legalconsideration.shtml https://transition.fec.gov/law/legalconsi
...
+rewrite ^/af/AFPRegulations.shtml https://www.fec.gov/legal-resources/enforcement/administrative-fines/#legal-citations redirect;
```
Comment (patphongs) [inline redirects-www.conf:137]: Remove, this is a duplicate

File: redirects-www.conf [hunk 6]
```diff
@@ -127,10 +127,16 @@ rewrite ^/law/legalconsideration.shtml https://transition.fec.gov/law/legalconsi
...
+rewrite ^/af/pay.shtml https://www.fec.gov/legal-resources/enforcement/administrative-fines/#paying-a-fine redirect;
+rewrite ^/af/payment/index.shtml https://www.fec.gov/legal-resources/enforcement/administrative-fines/#paying-a-fine redirect;
```
Comment (patphongs) [inline redirects-www.conf:139]: Remove and add as Wagtail redirect

File: redirects-www.conf [hunk 7]
```diff
@@ -143,33 +149,32 @@ rewrite ^/pdf/eleccoll.pdf https://transition.fec.gov/pdf/eleccoll.pdf redirect;
 rewrite ^/pdf/candgui.pdf /resources/cms-content/documents/candgui.pdf redirect;
 rewrite ^/pdf/partygui.pdf /resources/cms-content/documents/partygui.pdf redirect;
 rewrite ^/pdf/nongui.pdf /resources/cms-content/documents/nongui.pdf redirect;
-rewrite ^/elecfil/electron.shtml https://transition.fec.gov/elecfil/electron.shtml redirect;
+rewrite ^/elecfil/electron.shtml /help-candidates-and-committees/filing-reports/electronic-filing/ redirect;
 rewrite ^/efiling /help-candidates-and-committees/filing-reports/electronic-filing/ redirect;
 rewrite ^/support /help-candidates-and-committees/filing-reports/electronic-filing/ redirect;
 rewrite ^/elecfil/updatelist.shtml https://transition.fec.gov/elecfil/updatelist.html redirect;
 rewrite ^/elecfil/updatelist.html https://transition.fec.gov/elecfil/updatelist.html redirect;
```
Comment (patphongs) [inline redirects-www.conf:156]: Remove and add Wagtail redirect

File: redirects-www.conf (hunk 2 above)
Comment (dorothyyeager) [reply redirects-www.conf:130]: Done

File: redirects-www.conf (hunk 4 above)
Comment (dorothyyeager) [reply redirects-www.conf:136]: Done

File: redirects-www.conf (hunk 5 above)
Comment (dorothyyeager) [reply redirects-www.conf:137]: Done.

File: redirects-www.conf (hunk 6 above)
Comment (dorothyyeager) [reply redirects-www.conf:139]: Done.

File: redirects-www.conf (hunk 7 above)
Comment (dorothyyeager) [reply redirects-www.conf:156]: Done.

**Pull request merged**